pyproject.toml

```diff
@@ -139,7 +139,7 @@ docs = [
 ]
 dynamodb = [
   "boto3",
-  "commons-codec",
+  "commons-codec @ git+https://github.com/crate/commons-codec.git@sql-with-params",
```

Comment:
Sure enough, this and other relevant spots need to be amended after running another release of commons-codec.